templates/includes/sidebar.html

```diff
@@ -1,4 +1,10 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li><a href="{% url "taxi:driver-detail" pk=user.id %}">User: {{ user.username }}</a></li>
```

Comment:
Ensure that the `driver-detail` URL pattern correctly handles the `pk` parameter as `user.id`. If the `Driver` model's primary key does not match the user's ID, this could lead to incorrect URL generation.